CHANGES.rst

```diff
@@ -1,10 +1,21 @@
 CHANGES
 =======
 
-2.2 (unreleased)
+3.0 (unreleased)
 ----------------
 
-- Nothing changed yet.
+Backwards incompatible CHANGES
+++++++++++++++++++++++++++++++
+
+- Drop support for Python 3.4.
```

Comment:
As Python 3.4 had its sunset > ½ a year ago I do not see this as breaking backwards compatibility, especially as you did not do anything which is no longer Python 3.4 compatible e.g. requiring newer versions of the dependencies or adding a ` python_requires` parameter to `setup.py` or else.

Reply:
I added a new PR #3